pyked/chemked.py

```diff
@@ -175,6 +175,7 @@ def get_dataframe(self, output_columns=None):
 
         valid_labels = [a.replace('_', ' ') for a in self.__dict__ if not a.startswith('__')]
         valid_labels.remove('datapoints')
+        valid_labels.remove(' properties')
```

Comment:
I would suggest moving this a few lines up and adding it to the list generator,

    valid_labels = [a.replace('_', ' ') for a in self.__dict__ if not (a.startswith('__') or a.startswith('_'))]

That way we can add other "private" properties